paasta_tools/check_chronos_jobs.py

```diff
@@ -46,11 +59,30 @@ def compose_monitoring_overrides_for_service(chronos_job_config, soa_dir):
     monitoring_overrides['check_every'] = '1m'
     monitoring_overrides['runbook'] = monitoring_tools.get_runbook(
         monitoring_overrides, chronos_job_config.service, soa_dir=soa_dir)
-    monitoring_overrides['realert_every'] = monitoring_tools.get_realert_every(
-        monitoring_overrides, chronos_job_config.service, soa_dir=soa_dir)
+
+    if 'realert_every' not in monitoring_overrides:
+        guessed_realert_every = guess_realert_every(chronos_job_config)
+        if guessed_realert_every is not None:
+            monitoring_overrides['realert_every'] = monitoring_tools.get_realert_every(
+                monitoring_overrides, chronos_job_config.service, soa_dir=soa_dir,
+                monitoring_defaults=lambda x: {'realert_every': guessed_realert_every}.get(x))
+        else:
+            monitoring_overrides['realert_every'] = monitoring_tools.get_realert_every(
+                monitoring_overrides, chronos_job_config.service, soa_dir=soa_dir)
     return monitoring_overrides
 
 
+def add_realert_status(sensu_output, realert_every_in_minutes):
+    if realert_every_in_minutes is None or realert_every_in_minutes == -1:
```

Comment:
Don't bother with this case. If we can't predict when it will realert, then don't say anything.